python/versions.bzl

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""The Python versions we use for the toolchains.
+"""
+
+_RELEASE_BASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download"
+
+def get_release_url(platform, python_version):
+    release_filename = TOOL_VERSIONS[python_version]["url"].format(
+        platform = platform,
+        python_version = python_version,
+        build = "static-install_only" if ("windows" in platform) else "install_only",
+    )
+    url = "/".join([_RELEASE_BASE_URL, release_filename])
+    return (release_filename, url)
+
+# When updating the versions and releases, run the following command to get
+# the hashes:
+#   bazel run //python/private:print_toolchains_checksums
+#
+# buildifier: disable=unsorted-dict-items
+TOOL_VERSIONS = {
+    "3.8.10": {
```

Comment:
Why are there two versions of python3.8?

Reply:
because some users need that. We want to ensure that this new API doesn't force users to change their current interpreter version, it ought to allow any version that we know how to fetch